### ISSUE TITLE
[suggestion] update for latest conda/python

### ISSUE THREAD

**Issue by zdk123** [liyu95/DeepSimulator#24]
First I'd like to say thanks for this excellent tool... the application and feature sets fits my needs perfectly.

However, to future proof your code you might want to consider upgrading your scripts. Using `source activate` and `source deactivate` for conda environments is no longer recommended since conda 4.4 (released December 2017). 
https://github.com/conda/conda/blob/a4c4feae404b2b378e106bd25f62cc8be15c768f/CHANGELOG.md#440-2017-12-20

I have a local fork with relevant fixes, if you are accepting PRs and can issue it. The changes are minor but breaking.

Additionally, python2.7 will be reaching end-of-life in 2020. You should seriously consider migrating the code to python 3.7, since I image many developers will not keeping libraries (like tensorflow, sklearn, ...) maintained for any python 2 version for much longer.
https://pythonclock.org/

**Comment by lykaust15** [comment]
Hi, thank you very much for your interest and reminder!

Yes, we accept PRs and look forward to any kind of contribution in any form. It would be great if you can contribute to this tool.

In terms of the python version, we totally agree with you. In the near future, we will provide support for both python2.7 and python3. 

Sincerely,
Yu

**Comment by zdk123** [comment]
see PR #26

**Comment by lykaust15** [comment]
Hi, thank you very much for your great contribution!
I checked the updated script with different environments. On conda4.4, it works perfectly, which is great! However, on conda3, I got some errors. To remain the stability of the master branch, I created a new branch 'conda_newest' and accepted the PR to that branch. Later on, I will consider merging the change to the master branch.

Thank you again for your contribution!

**Comment by zdk123** [comment]
Yes, I would expect conda 3 to fail, and I'm not sure how easy it would be to maintain backwards compatibility.  But I'm not sure you should want to - the conda 3 release was in 2016!

**Comment by nick-youngblut** [comment]
The README still states to use anaconda/miniconda v2, and the python version in the `install.sh` script is still python v2. I'm curious why this issue is closed. 

As with @zdk123, I'm worried about the longevity of this software if it remains dependent and python2 (or at least still recommends python=2.7 and older versions of other dependencies). I just found DeepSimulator and was considering using it, but I'm inclined to use NanoSim-H, given that it is python3 compatible and has a bioconda recipe for simple installation and version management.

**Comment by liyu95** [comment]
> The README still states to use anaconda/miniconda v2, and the python version in the `install.sh` script is still python v2. I'm curious why this issue is closed.
> 
> As with @zdk123, I'm worried about the longevity of this software if it remains dependent and python2 (or at least still recommends python=2.7 and older versions of other dependencies). I just found DeepSimulator and was considering using it, but I'm inclined to use NanoSim-H, given that it is python3 compatible and has a bioconda recipe for simple installation and version management.

Dear Nick:

Thank you very much for the interest!
This repo was not well-maintained mainly because I was in a transition state from a Ph.D. student to an AP. I did not have too much time to maintain it. As now I am getting stable, I will identify some students to maintain this tool and further increase more functionality to it.

**Comment by nick-youngblut** [comment]
That's great to hear! Thanks for the quick feedback
